### DISCUSSION TITLE
Improved Map Voting

### DISCUSSION THREAD

**Discussion by NotAFile** [piqueserver/piqueserver#640] [Feature Requests]
often people don't like the map rotation, /advancemap votes might be good for that

**Comment by 1AmYF** [comment]
This allows map votes https://github.com/piqueserver/piqueserver/blob/master/piqueserver/config/scripts/votemap.py
I never used it though, i think it's too inconvenient to handle as players will need to know the mapnames.

**Comment by noway** [comment]
That might be where advanced GUI comes in. If you can draw boxes and layouts on play screen, they can see the map list for example, and select their choice by simply clicking 1,2,3,4.
